app/models/book.rb

```diff
@@ -120,7 +120,6 @@ def sync_keywords
     end
 
     def format_fields
-      self.isbn = self.isbn.tr('-','') if isbn.present?
+      self.isbn = self.isbn.tr('-', '') if isbn.present?
```

Comment:
Redundant self detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.